source/includes/steps-fle-configure-the-mongodb-client.yaml

```diff
@@ -87,55 +87,55 @@ content: |
   MongoDB drivers use the ``mongocryptd`` binary to perform client-side
   encryption. For automatic encryption, the client manages the
   ``mongocryptd`` process. To configure the client to find the binary,
-  MedcoMD specifies the binary's path as an extra option:
-  
+  Specify the path to the binary using the following configuration:
+
   .. tabs-drivers::
-   
+
      .. tab::
         :tabid: java-sync
-        
+
         .. code-block:: java
            :emphasize-lines: 2
-           
+
            final Map<String, Object> extraOptions = new HashMap<String, Object>() {{
-             put("mongocryptdPath", "/usr/local/bin/mongocryptd");
+             put("mongocryptdSpawnPath", "/usr/local/bin/mongocryptd");
            }};
-        
+
         .. admonition:: Encryption Binary Daemon
            :class: note
-        
+
            If the ``mongocryptd`` daemon is already running, you can
            configure the client to skip starting it by passing the
            following option:
-           
+
            .. code-block:: java
               :emphasize-lines: 2
-              
+
               final Map<String, Object> extraOptions = new HashMap<String, Object>() {{
                 put("mongocryptdBypassSpawn", true);
               }};
      .. tab::
         :tabid: nodejs
-        
+
         .. code-block:: javascript
            :emphasize-lines: 2
-           
+
            const extraOptions = {
-            mongocryptdPath: "/usr/local/bin/mongocryptd";
+             mongocryptdPath: "/usr/local/bin/mongocryptd",
```

Comment:
I'm sorry this one is probably my error, but the extraOptions takes in a key called "mongocryptdSpawnPath" not "mongocryptdPath". See: https://www.npmjs.com/package/mongodb-client-encryption#AutoEncrypter..AutoEncryptionExtraOptions

Reply:
Will update